src/views/forms/AccountManagementForms/WithdrawForm.tsx

```diff
@@ -159,6 +161,8 @@ export const WithdrawForm = () => {
 
   const onSubmit = useCallback(
     async (e: FormEvent) => {
+      const notificationId = Date.now().toString();
```

Comment:
do we have a `uuid` package in this repo? probably better than using `Date.now()` considering it's technically possible to fire 2 notifications at the same time which could cause some major headaches

Reply:
I can change to uuid, with timestamp as fallback.